src/components/atoms/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import { GlobalChildren } from "components/templates/GlobalChildren.interface";
+
+interface errorInterface extends GlobalChildren {
+    errorArray: Array<string>;
+}
+
+const ErrorMessages = ({ errorArray }: errorInterface) => {
```

Comment:
```suggestion
const ErrorMessage = ({ errorArray }: errorInterface) => {
```
Le nom du fichier est ErrorMessage donc soit changer le nom du fichier, soit changer le nom de l'export ou du composant.
Avec ou sans S à ton choix...

Reply:
!reminder @AntoineCadoret